src/lib.rs

```diff
@@ -155,6 +155,7 @@ use std::sync::atomic::{AtomicBool, AtomicU32, Ordering};
 use std::sync::{Arc, Once};
 use std::{cmp, mem, ptr, slice, thread};
 
+use in_range::InRange;
```

Comment:
```suggestion
use crate::in_range::InRange;
```

I thought this was enforced by `rustfmt`, but could you make imports from `rav1d` use `crate::`?  That way it's grouped with all of the other `use crate::`s.